source/postcard-schema/src/impls/builtins_std.rs

```diff
@@ -21,6 +21,14 @@ impl Schema for std::string::String {
     };
 }
 
+#[cfg_attr(docsrs, doc(cfg(any(feature = "alloc", feature = "use-std"))))]
+impl Schema for std::path::PathBuf {
+    const SCHEMA: &'static NamedType = &NamedType {
+        name: "PathBuf",
+        ty: &DataModelType::UnitStruct,
```

Comment:
This is probably not correct, do you know how PathBuf actually gets serialized? Is it serialized as a string?

It appears so: https://github.com/serde-rs/serde/blob/master/serde/src/ser/impls.rs#L957-L980

In general, the `ty` needs to match exactly how the type is actually serialized/deserialized, otherwise libraries like `postcard-dyn` won't work correctly.

Reply:
If I say `ty: &DataModelType::String`, `to_pseudocode` will print `String` for `PathBuf`. If that's all right, I'm happy to change it.

Reply:
Yeah, the alternative would be to make this a `TupleStruct`, with `String` as the inner type, and "PathBuf" as the name of the outer struct.

I'd have to check if we do that anywhere else for consistency, but I'm less fussed about that as that is more aesthetic than functional.

Reply:
That's to be expected since `Serialize`/`Deserialize` treats `PathBuf` and `String` in the exact same way

Reply:
Ah, sorry, probably `NewTypeStruct`, not `TupleStruct`.

Reply:
> Yeah, the alternative would be to make this a `TupleStruct`, with `String` as the inner type, and "PathBuf" as the name of the outer struct.
> 
> I'd have to check if we do that anywhere else for consistency, but I'm less fussed about that as that is more aesthetic than functional.

`NewTypeStruct(T)` vs `T` may be a functional difference as far as `postcard-dyn` is concerned